modules/nf-core/ensemblvep/vep/main.nf

```diff
@@ -59,9 +58,9 @@ process ENSEMBLVEP_VEP {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.ann.vcf.gz
-    touch ${prefix}.ann.tab.gz
-    touch ${prefix}.ann.json.gz
+    touch ${prefix}.vcf.gz
```

Comment:
no updates in the modules.config to match the naming scheme?

Reply:
that's for the stub

Reply:
🤦‍♀️